package.json

```diff
@@ -11,14 +11,20 @@
     "denali-addon",
     "addon"
   ],
+  "main": "app/addon.js",
```

Comment:
@knownasilya Sounds good on the `index.js` file (and thanks, I missed that on the blueprint update).

Regarding `dist` being there in the blueprint, what I ran into when running this with the `ember-help-wanted-backend` app is that the addition of `dist` (in `dist/app/addon.js`) combined with https://github.com/denali-js/denali/blob/master/lib/runtime/addon.js#L161 meant that the addon couldn't be loaded by Denali.  Hence the change 😀 

So how do we want to handle this?  Unless I set up the app wrong (a possible scenario in all of this) the double dist path that Denali looks for (which would match blueprints) would lead to in inoperable add-on 😀